src/command_modules/azure-cli-network/azure/cli/command_modules/network/mgmt_vpn_connection/azuredeploy.json

```diff
@@ -0,0 +1,118 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkGatewayConnectionName": {
+      "type": "string",
+      "metadata": {
+        "description": "Connection name."
+      }
+    },
+    "connectionType": {
+      "type": "string",
+      "allowedValues": [
+        "IPSec",
+        "Vnet2Vnet",
+        "ExpressRoute"
+      ],
```

Comment:
this is a template switch variable, so it can't have arbitrary values

---
In reply to: [74603489](https://github.com/Azure/azure-cli/pull/687#discussion_r74603489) [](ancestors = 74603489)